demos/JWST/S4_template.ecf

```diff
@@ -9,6 +9,7 @@ wave_max        4.5         # Maximum wavelength. Set to None to use the longest
 allapers        False       # Run S4 on all of the apertures considered in S3? Otherwise will use newest output in the inputdir
 
 # Parameters for drift correction of 1D spectra
+recordDrift     False    # Set True to record drift/jitter in 1D spectra (always recorded if correctDrift is True)
```

Comment:
Please document this new ECF setting at docs/source/ecf.rst

Reply:
done